extensions/wikia/Search/WikiaSearchAjaxController.class.php

```diff
@@ -14,59 +16,60 @@ class WikiaSearchAjaxController extends WikiaController {
 	/**
 	 * Handles accessing paginated results via AJAX.
 	 */
-	public function getNextResults(){
-		wfProfileIn(__METHOD__);
+	public function getNextResults() {
 
-		$this->response->setVal('status', true);
+		$this->response->setVal( 'status', true );
 
-		$query = urldecode( $this->request->getVal('query', $this->request->getVal('search')) );
+		$query = urldecode( $this->request->getVal( 'query', $this->request->getVal( 'search' ) ) );
 
-		$page = $this->request->getVal('page', 1);
-		$rank = $this->request->getVal('rank', 'default');
-		$crossWikia = $this->request->getBool('crossWikia');
+		$page = $this->request->getVal( 'page', 1 );
+		$rank = $this->request->getVal( 'rank', 'default' );
+		$crossWikia = $this->request->getBool( 'crossWikia' );
 		$hub = false;
 
-		$isCorporateWiki = !empty($this->wg->EnableWikiaHomePageExt);
+		$isCorporateWiki = !empty( $this->wg->EnableWikiaHomePageExt );
 
 		$isInterWiki = $crossWikia ? true : $isCorporateWiki;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "true" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)